shallow_backup/utils.py

```diff
@@ -12,10 +12,10 @@ def run_cmd(command):
 	"""
 	try:
 		if not isinstance(command, list):
-			process = sp.run(command.split(), stdout=sp.PIPE)
+			process = sp.run(command.split(), stdout=sp.PIPE, stderr=sp.DEVNULL)
```

Comment:
```suggestion
			process = sp.run(command.split(), stdout=sp.PIPE)
```

Reply:
The `stderr` redirection is to prevent installed but failed commands from showing their errors.

Reply:
Wouldn't it also hide any fatal errors thrown by a package manager?

Reply:
Yeah but that's why we did the `An error occurred while running blah blah` because otherwise the regular traceback would kinda ruin the pretty little prompts we got going.

I think finding a way to suppress but capture `stderr` would be best but I think that should be a separate issue and this should work fine for now.

But if you want we could show the errors

Reply:
> I think finding a way to suppress but capture stderr would be best but I think that should be a separate issue and this should work fine for now.

Alright with me, wanna open a new issue?

Reply:
> also I don't think any of the package commands would fail I mean they all are just like using their list function.

You can get some *weird* errors if you fuck with permissions the wrong way for important files for package managers. On a system where everything is working well, you're right. 

Check this [one out.](https://stackoverflow.com/questions/10580813/how-do-i-fix-brew-doctor-errors)

Reply:
ah gotcha, that's a good point. Yeah I think capturing the traceback would definitely be the best way to go. I'll go make the issue right after cleaning up the PR

Reply:
Copying here for reference later: 

> I've been thinking about this some more and I'm a bit concerned about hiding all the stderr output. If I were a user and the shell commands fail, I'd like them to fail loudly and give me the error readout right in the terminal. I like handling errors for shallow-backup cleanly, but this would be hiding non-shallow-backup errors.

Reply:
The [issue's](#208) up.

I'm probably gonna take a break from working on it to get some ground on a project of my own but I'll try to think of something for it  😄

Reply:
> I'm probably gonna take a break from working on it to get some ground on a project of my own

Absolutely. It's been a pleasure working with you on this PR!